src/pages/SplitPosition/Form.test.tsx

```diff
@@ -65,35 +65,6 @@ test('show unlock button with zero allowance', async () => {
   expect(unlockBtn).toBeInTheDocument()
 })
 
-test('toggle unlock button visiblity according to allowance and amount', async () => {
```

Comment:
This test, was not update since a long time, and throw an error, check for some stuff that doesn't exist

Reply:
In the same field, we can see some lines commented , so ...